### PR TITLE
Revert useless alpha check in jsb-action

### PR DESCRIPTION
**Pull request by pandamicro**
Re: cocos-creator/fireball#6163

Changes proposed in this pull request:
 * Revert useless alpha check in jsb-action

> Makes sure these boxes are checked before submitting your PR - thank you!
>
- [x] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
- [x] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.
- To official teams:
  - [x] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
  - [x] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)
  - [x] Make sure any runtime log information in `cc.log` , `cc.error`, `cc.warn` or `cc.assert` has been moved into `DebugInfos.js` with an ID

@cocos-creator/engine-admins

### CODE REVIEW

File: jsb/jsb-action.js [hunk 1]
```diff
@@ -469,11 +469,6 @@ function syncColorUpdate (dt) {
     var target = this._getSgTarget();
     if (target._owner) {
         var color = target.getColor();
-        var alpha = color.a;
-        if (alpha !== 255) {
-            target._owner.opacity = alpha;
-            color.a = 255;
```
Comment (jareguo) [inline jsb/jsb-action.js:475]: 这个重置 a 的操作去掉后，模拟器不就会有警告？

Comment (pandamicro) [reply jsb/jsb-action.js:475]: 这个 PR 会修复 https://github.com/cocos-creator/cocos2d-x-lite/pull/730

Comment (pandamicro) [reply jsb/jsb-action.js:475]: 之前的警告是因为 color.a 为 undefined

Comment (jareguo) [reply jsb/jsb-action.js:475]: 哦~

**Pull request merged**